__DEFINES/setup.dm

```diff
@@ -277,6 +277,30 @@ var/MAX_EXPLOSION_RANGE = 14
 #define slot_legcuffed 17
 #define slot_legs 18
 
+//slots, but in string format.
+#define slot_back_str "1"
```

Comment:
why aren't these in capslock too?

Reply:
Mostly because `slot_back` and whatever are not capitalised. And they're used a lot, so I believe making them all caps would hurt readability rather than improve it.